### PR TITLE
Sdil 615

### PR DESCRIPTION
**Pull request by mattrobertsky**
This will be a bit of a thorny merge but I don't mind doing it. And I'll tidy up the messages once we've merged as namespaces are not uniform right now and we will have duplicated messages.

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+GET         /assets/*file                  controllers.Assets.at(path="/public", file)
 
-GET         /copacked-produced        controllers.LiabilityToolController.displayCopackedProduced
-POST        /copacked-produced        controllers.LiabilityToolController.submitCopackedProduced
-GET         /copacked-commissioned    controllers.LiabilityToolController.displayCopackedCommissioned
-POST        /copacked-commissioned    controllers.LiabilityToolController.submitCopackedCommissionedForm
+GET         /hello-world                   controllers.LiabilityToolController.helloWorld()
+
+GET         /manufacture-who               controllers.LiabilityToolController.whoDoYouManufactureForPage()
+POST        /submit-manufacture-who        controllers.LiabilityToolController.submitWhoDoYouManufactureForPage()
```
Comment (nkaraolis) [inline conf/app.routes]: Any reason the submit URL is different to the page URL? I don't mind but we should decide what format we are going with to be consistent

Comment (nkaraolis) [reply conf/app.routes]: Oh and run an autoformat on this file please :nerd_face:

File: conf/application.conf [hunk 2]
```diff
@@ -1,4 +1,5 @@
-appName = "soft-drinks-industry-levy-liability-tool-frontend"
+#appName = "soft-drinks-industry-levy-liability-tool-frontend"
```
Comment (nkaraolis) [inline conf/application.conf]: Can we keep this as the same as the repository name as this is used in the logging config and makes it easier for us to find our app. If you want to use the other name can you load it from the messages instead

File: test/controllers/LiabilityToolControllerSpec.scala [hunk 3]
```diff
@@ -13,9 +14,11 @@ class LiabilityToolControllerSpec extends PlaySpec with MockitoSugar with GuiceO
 
   val messagesApi: MessagesApi = app.injector.instanceOf[MessagesApi]
   val mockConfig: Configuration = app.injector.instanceOf[Configuration]
+  val backendConnector: BackendConnector = mock[BackendConnector]
```
Comment (nkaraolis) [inline test/controllers/LiabilityToolControllerSpec.scala]: Remove this as no longer required

File: test/controllers/LiabilityToolControllerSpec.scala [hunk 4]
```diff
@@ -13,9 +14,11 @@ class LiabilityToolControllerSpec extends PlaySpec with MockitoSugar with GuiceO
...
 
-  val mockLiabilityController = new LiabilityToolController(messagesApi, mockConfig)
-
+  val mockLiabilityToolController = new LiabilityToolController(messagesApi, backendConnector, mockConfig)
```
Comment (nkaraolis) [inline test/controllers/LiabilityToolControllerSpec.scala]: Remove as no longer required

File: app/controllers/LiabilityToolController.scala [hunk 5]
```diff
@@ -10,10 +10,67 @@ import play.api.i18n.{I18nSupport, Messages, MessagesApi}
...
+      formWithErrors => {
+        BadRequest(liability(formWithErrors))
+      },
+      validFormData => {
+        Redirect(routes.LiabilityToolController.helloWorld()) // TODO replace when we know the page flow
+      }
+    )
+  }
+
+  def errorCheck(tuple2: Tuple2[Option[Boolean], Option[Boolean]]): Boolean = {
```
[the remark this answers is not in the document]
Comment (nkaraolis) [reply app/controllers/LiabilityToolController.scala]: Also you can neaten the Tuple slightly by saying (Option[Boolean], Option[Boolean])

File: app/views/pages/who_do_you_manufacture_for.scala.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
+@import play.api.Configuration
+@import uk.gov.hmrc.play.views.html.{helpers => govHelpers}
+@import helpers.checkbox
+
+@(whoForFormTuple: Form[Tuple2[Option[Boolean], Option[Boolean]]])(implicit messages: Messages, request: Request[_], configuration: Configuration)
```
Comment (nkaraolis) [inline app/views/pages/who_do_you_manufacture_for.scala.html]: You can neaten the form declaration to this if you wanted to Form[(Option[Boolean], Option[Boolean])]

File: test/controllers/LiabilityToolControllerSpec.scala [hunk 7]
```diff
@@ -13,9 +14,11 @@ class LiabilityToolControllerSpec extends PlaySpec with MockitoSugar with GuiceO
...
+  
+  // these should be moved to the package object
```
Comment (nkaraolis) [inline test/controllers/LiabilityToolControllerSpec.scala]: Can we move these last few forms to the package object and remove this comment please

File: app/controllers/LiabilityToolController.scala [hunk 8]
```diff
@@ -10,10 +10,67 @@ import play.api.i18n.{I18nSupport, Messages, MessagesApi}
 import play.api.mvc.{Action, AnyContent, Controller}
 import views.html.pages._
 
+
 @Singleton
-class LiabilityToolController @Inject()(val messagesApi: MessagesApi,
+class LiabilityToolController @Inject()(val messagesApi: MessagesApi, backendConnector: BackendConnector,
```
[the remark this answers is not in the document]
Comment (mattrobertsky) [reply app/controllers/LiabilityToolController.scala]: Sure.

File: app/controllers/LiabilityToolController.scala (hunk 5 above)
[the remark this answers is not in the document]
Comment (mattrobertsky) [reply app/controllers/LiabilityToolController.scala]: Sure.

File: conf/app.routes (hunk 1 above)
Comment (mattrobertsky) [reply conf/app.routes]: Yep.

File: conf/application.conf (hunk 2 above)
Comment (mattrobertsky) [reply conf/application.conf]: Cool

File: test/controllers/LiabilityToolControllerSpec.scala (hunk 4 above)
Comment (mattrobertsky) [reply test/controllers/LiabilityToolControllerSpec.scala]: Done

File: test/controllers/LiabilityToolControllerSpec.scala (hunk 7 above)
Comment (mattrobertsky) [reply test/controllers/LiabilityToolControllerSpec.scala]: Done

File: app/views/pages/who_do_you_manufacture_for.scala.html (hunk 6 above)
Comment (mattrobertsky) [reply app/views/pages/who_do_you_manufacture_for.scala.html]: Done

**Pull request merged**